assets/scss/components/stencil/applePay/_applePay.scss

```diff
@@ -0,0 +1,54 @@
+$apple-pay-black: black;
+$apple-pay-white: white;
+
+.apple-pay-checkout-button {
+    background-position: 50% 50%;
+    background-repeat: no-repeat;
+    background-size: 100% 60%;
+    border-radius: 4px;
+    box-sizing: border-box;
```

Comment:
I think this should be set by default already (unless we reset it for some reason)